features/git-town/configuration/configuration.feature

```diff
@@ -9,28 +9,54 @@ Feature: listing the configuration
     Given I have configured the main branch name as "main"
     And my non-feature branches are "qa" and "staging"
     When I run `git town config`
-    Then I see "Main branch: main"
-    And I see "Non-feature branches: qa, staging"
+    Then I see
+      """
+      Main branch:
+      main
```

Comment:
I like that we output only `main` if I ask specifically for the main branch name. But this output feels like a step backwards. 

```
Main branch: main
```

reads a lot better to me than

```
Main branch:
main
```